### DISCUSSION TITLE
Tracking flies in a video in the dark

### DISCUSSION THREAD

**Discussion by akshitasax** [talmolab/sleap#774] [Help!]
Hi,

I want to track the movement of flies at night. I recorded a 12 hour long video of 2 flies at night, and used SLEAP to train a model after labeling 20 frames. I tried predicting on 20 random frames, and this was the result for almost all the frames that were predicted on (screenshot below). Is this a problem of the video quality or lighting condition (I can try to introduce infrared lighting), or is there a way to reliably track this in SLEAP as it is?

![Screenshot from 2022-06-02 15-53-27](https://user-images.githubusercontent.com/104799583/171751201-2608a040-2c6b-431e-8861-ef65ea6b48b7.png)

these were the settings used for the training pipeline
![Screenshot from 2022-06-02 15-55-00](https://user-images.githubusercontent.com/104799583/171751364-adecfeff-0d8d-4fb4-9674-bfc42f64f7cd.png)

Thank you,
Akshita

(I already have a discussion thread open for a different issue but thought I'd make this a new topic)

**Comment by roomrys** [comment]
Hi @akshitasax,

Those are definitely not the type of predictions we want to see. 

Can you run the `sleap-inspect` command on your project and paste the output? Also do you mind sending a screenshot of the videos tab (in case this is an issue related to the video backend used).

Thanks,
Liezl
